app/services/case_require_further_action_service.rb

```diff
@@ -15,7 +15,7 @@ def initialize(user, kase, update_parameters)
   def call
     begin
       ActiveRecord::Base.transaction do
-        update_attributes
+        update
```

Comment:
Same here, false positive, this is calling a custom method defined below, `#update_attributes`

Reply:
Probably better to rename the method to `do_update` or anything like that so this can't happen in the future 😬